src/herder/test/HerderTests.cpp

```diff
@@ -606,7 +605,7 @@ testTxSetWithFeeBumps(uint32 protocolVersion)
             auto tx1 = transaction(*app, account1, 1, 1, 100);
             auto fb1 = feeBump(*app, account2, tx1, 200);
             txSet->add(fb1);
-            auto tx2 = transaction(*app, account2, 1, 1, -1);
+            auto tx2 = transaction(*app, account2, 1, -1, 100);
```

Comment:
same thing here re: "malformed", I think you want to keep "1" for the sequence number increment